pkg/didcomm/common/service/service.go

```diff
@@ -30,6 +45,33 @@ type DIDCommMsg struct {
 	Payload  []byte
 	// TODO : might need refactor as per the issue-226
 	OutboundDestination *Destination
+	Common              *Common
```

Comment:
The type field is now duplicated (above).

Reply:
Done!